includes/polling/ports.inc.php

```diff
@@ -224,11 +210,11 @@
     }
 }
 
-if ($config['enable_ports_etherlike']) {
-    echo 'dot3Stats ';
-    $port_stats = snmpwalk_cache_oid($device, 'dot3StatsEntry', $port_stats, 'EtherLike-MIB');
-} elseif ($device['os'] != 'asa') {
+if ($device['os'] != 'asa') {
     echo 'dot3StatsDuplexStatus';
+    if ($config['enable_ports_poe'] || $config['enable_ports_etherlike']) {
+        $port_stats = snmpwalk_cache_oid($device, 'dot3StatsIndex', $port_stats, 'EtherLike-MIB');
```

Comment:
We use only this + dot3StatsDuplexStatus within poe and etherlike port polling files so only look those up.